orx-compositor/src/commonMain/kotlin/Compositor.kt

```diff
@@ -124,6 +131,14 @@ open class Layer internal constructor() {
                 }
             }
 
+            val colorBuffer = if (bufferMultisample == BufferMultisample.Disabled) {
+                target.colorBuffer(0)
+            } else {
+                val resolved = colorBuffer(target.colorBuffer(0).width, target.colorBuffer(0).height)
```

Comment:
Looks like a colorBuffer is allocated per draw and nobody is cleaning this up?

Reply:
oops, I did not know any cleaning had to be done 😅